indico/htdocs/js/indico/modules/contributions/common.js

```diff
@@ -105,48 +105,56 @@
         });
     }
 
+    function markContributionAsUnscheduled(itemPicker) {
```

Comment:
I'll have to rename this later since undo may set a new date. Anyway, I can do that once I add the undo logic again.